docs/src/upgrade/release-3.7.x.asciidoc

```diff
@@ -29,6 +29,28 @@ Please see the link:https://github.com/apache/tinkerpop/blob/3.7.0/CHANGELOG.asc
 
 === Upgrading for Users
 
+==== Properties on Elements
```

Comment:
i can't recall, but are there any compatibility issues to call attention to here? like can a 3.6.x driver work nicely with 3.7.x or the reverse? maybe there is something to call attention to here with that.

Reply:
different behavior for different GLV's
Python, Go: deserialization error for GraphBinary, skip properties for GraphSON
.NET:  skip properties
Javascript: should work ok

also serialization can't be managed with `materializeProperties` option

Reply:
your recent work shows `ReferenceElement`-type objects are no longer returned - you get a `DetachedElement` from remote requests. Folks may not have been coding to the `Element` interfaces as they likely should have been and therefore could been working directly with "reference" implementations. I think upgrade docs should point out the change and the appropriate way to avoid problems in the future - i.e. use the interfaces like `Vertex` and `Edge`.